features/register.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Register
+  In order to sign up
+  As a user
+  I need to be able to fill in the data correctly and click Register
+
+  Scenario: Registration with correct data
```

Comment:
Very important!
This causes side effect (new user in database). You should add @database (for example) tag and implement AfterScenario hook in FeatureContext.